lib/winston-logstash-udp.js

```diff
@@ -46,7 +46,13 @@ util.inherits(LogstashUDP, winston.Transport);
 winston.transports.LogstashUDP = LogstashUDP;
 
 LogstashUDP.prototype.connect = function() {
+    var self = this;
     this.client = dgram.createSocket('udp4');
+
+    this.client.on('error', function(err) {
+        // Skipping DNS errors
+        self.emit('error', err);
```

Comment:
This is not the only error i have. With different version of OS, i have these error code ESRCH (centos7), EAI_AGAIN (ubuntu), etc...
I think it depend on how dns lookup is manage on OS